frappe/desk/notifications.py

```diff
@@ -279,17 +279,9 @@ def get_open_count(doctype, name, items=None):
 			# get the fieldname for the current document
 			# we only need open documents related to the current document
 			filters[fieldname] = name
-			total = len(
-				frappe.get_all(d, fields="name", filters=filters, limit=100, distinct=True, ignore_ifnull=True)
```

Comment:
This is kinda incorrect:
1. db.count - doesn't support linking child table like we do in `get_all`. Open Sales Invoice doctype and see, this wont work.
2. Alternate is using `get_all(field="count(*)")` but that won't give distinct count without adding `group by` for parent's name.

Reply:
Damn, I missed that, sorry 😞

The second approach probably won't result in much speedup. Figuring out if we're dealing with a child table and need to group will also takes it's time. Unless there's a fast way to do this.

Reply:
All the benefit here is "not reading and sending document names" so if we get count out of DB directly it's still fast. 

Lets trust query planners to do the necessary work :woozy_face: